cloud-functions/functions/custom-logo.js

```diff
@@ -0,0 +1,50 @@
+// this is meant to be run by Covid Watch devs to set a custom logo for different organizations to appear in the NavBar
```

Comment:
Let's call this something like `super-admin.js` so it makes sense to add other super admin functions to it as needed.